docs/ciphers.rst

```diff
@@ -127,15 +124,15 @@ which uses the ChaCha and Poly1305 algorithms, and which is already
 implemented by the Chrome browser.  Mozilla has delayed recommending
 ``0xcc13`` over compatibility and standardization concerns, but is likely
 to recommend it in the future once these concerns have been addressed. At
-that point, the Let's Encrypt client would likely follow the Mozilla
-recommendations and favor the use of this ciphersuite as well.)
+that point, Certbot would likely follow the Mozilla recommendations and favor
+the use of this ciphersuite as well.)
 
 The Let's Encrypt project may deviate from the Mozilla recommendations
```

Comment:
You mean this paragraph and the previous? I think Certbot is more accurate here.